engines/testbed/config.cpp

```diff
@@ -27,72 +27,107 @@
 #include "testbed/config.h"
 #include "testbed/fs.h"
 
+#include "gui/widgets/scrollcontainer.h"
+#include "gui/gui-manager.h"
+#include "gui/ThemeEval.h"
+
 namespace Testbed {
 
 TestbedOptionsDialog::TestbedOptionsDialog(Common::Array<Testsuite *> &tsList, TestbedConfigManager *tsConfMan) :
 		GUI::Dialog("TestbedOptions"),
```

Comment:
This needs to be removed as well, and the dialog dimensions calculated instead., e.g. write here  `GUI::Dialog(0, 0, 0, 0)` and then start calculating the dialog dimensions.